HUD/SetNotificationMessage.md

```diff
@@ -18,7 +18,7 @@ Example result:
 
 Old description with list of possible icons and texture names:
 
-```text
```

Comment:
Was this really necessary?

Reply:
It was causing the old twing template to blow-up, at least when I deployed it on my end.